Tools/Scripts/webkitpy/benchmark_runner/browser_driver/osx_safari_driver.py

```diff
@@ -1,8 +1,9 @@
 import logging
 import os
-import sys
 import re
+import shutil
```

Comment:
unused import?

Reply:
Indeed, thanks!

Reply:
Wait, but I see `shutil.copytree` below?

Reply:
…in `benchmark_runner.py`, not this file. (This import is in fact because I was originally making the `copytree` call from the driver)